shopping_list/src/main/java/com/zizohanto/android/tobuy/shopping_list/ui/shopping_list/adaper/ShoppingListAdapter.kt

```diff
@@ -100,7 +99,8 @@ class ShoppingListAdapter @Inject constructor() :
                     oldItem: ShoppingListWithProductsModel,
                     newItem: ShoppingListWithProductsModel
                 ): Boolean {
-                    return oldItem.shoppingList == newItem.shoppingList
+                    return oldItem.products.size == newItem.products.size &&
```

Comment:
Why are we also checking the size of the products?

Reply:
that's the fix

Reply:
that fixes the issue linked to the PR

Reply:
Oh, seen.